src/app/user/user.service.ts

```diff
@@ -1,8 +1,20 @@
+import { Response, Request } from 'express';
+
+import { IUser } from '../user/user.type';
+
 import User from './user.model';
+
 export class UserService {
+
   private model = User;
 
-  getUser(): string {
-    return '';
+  async getUser(req: Request, res: Response): Promise<void> {
```

Comment:
@hsedaqat1378 also you can accept filters for the user so pass some input like `filters` and this filter could be some properties that user has and we can pass it to `where` condition on the query.

Reply:
i not sure how change this but I'm try to better, Thanks a lot man